lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +193,66 @@ class RenderForm extends React.Component {
     );
   }
 
+  async handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (queryString.length > 2) {
+      let suggestions = [];
+      await fetch(`${this.learnerSupportCenterURL}/hc/api/internal/instant_search.json?query=${queryString}&locale=en-us`, { headers: {
+        'Content-Type': 'application/json',
+      }}).then((response) => {
+        suggestions = response.json.results;
+        this.setState({ suggestions });
+      });
+    } else {
+      this.setState({
+        suggestions: [],
+        activeSuggestion: 0,
+      });
+    }
+  }
+
+  onKeyDown(event) {
+    const { activeSuggestion, suggestions } = this.state;
+
+    if (event.keyCode === 13) { // User pressed the enter key
+      window.location.href = this.learnerSupportCenterURL + suggestions[activeSuggestion].url;
+    } else if (event.keyCode === 38) { // User pressed the up arrow
+      if (activeSuggestion === 0) { return; }
+
+      this.setState({ activeSuggestion: activeSuggestion - 1 });
+    } else if (event.keyCode === 40) { // User pressed the down arrow
+      if (activeSuggestion - 1 === suggestions.length) { return; }
```

Comment:
same question. If the current item is the last, pressing down should take the user to the first item. This will remove this unnecessary if conditions. You can use max/min to decide the value of activeSuggestion